plugins/README.md

```diff
@@ -66,7 +66,6 @@ DB copy provides a helper which can copy LevelDB or RocksDB data quickly on the
   java -jar Toolkit.jar db cp  output-directory/database /tmp/databse
 ```
 
-
 ## DB Lite
 
 DB lite provides lite database, parameters are compatible with previous `LiteFullNodeTool`.
```

Comment:
LiteFullNodeTool has been removed ,is it reasonable to quote  something not existed?

Reply:
This description is an indication to reference the previous `LiteFullNodeTool` instead of using the `LiteFullNodeTool`